test/e2e/network/service.go

```diff
@@ -124,24 +124,15 @@ var _ = SIGDescribe("Services", func() {
 	framework.ConformanceIt("should serve a basic endpoint from pods ", func() {
 		serviceName := "endpoint-test2"
 		ns := f.Namespace.Name
-		jig := e2eservice.NewTestJig(cs, serviceName)
-		labels := map[string]string{
-			"foo": "bar",
-			"baz": "blah",
-		}
+		jig := e2eservice.NewTestJig(cs, ns, serviceName)
 
 		ginkgo.By("creating service " + serviceName + " in namespace " + ns)
 		defer func() {
 			err := cs.CoreV1().Services(ns).Delete(serviceName, nil)
 			framework.ExpectNoError(err, "failed to delete service: %s in namespace: %s", serviceName, ns)
 		}()
-		ports := []v1.ServicePort{{
-			Port:       80,
-			TargetPort: intstr.FromInt(80),
-		}}
-		_, err := jig.CreateServiceWithServicePort(labels, ns, ports)
-
-		framework.ExpectNoError(err, "failed to create service with ServicePorts in namespace: %s", ns)
+		_, err := jig.CreateTCPServiceWithPort(nil, 80)
```

Comment:
Why delete `CreateServiceWithServicePort()`?
Also, it looks like we are not building the service with the labels `foo=bar` and `baz=blah`.